.github/workflows/links.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   linkChecker:
+    permissions: write-all
```

Comment:
Let's scope down the permissions so we only grant issues write access. I think this should suffice:

```
permissions:
  issues: write
```
nit: instead of doing it inline at the job level, I'd do it at the top level `permissions` , such as this: https://github.com/aptos-labs/aptos-core/blob/f1ca3ef086472fcea572657e263d49b812c97b35/.github/workflows/close-stale-issues.yaml#L7-L10

Reply:
Thanks, Rustie! I believe I addressed this. Please take another look.